src/comparator/wrappers.py

```diff
@@ -219,19 +219,20 @@ def required(self):
 
     @property
     def proto_type(self):
-        """Return the proto type constant e.g. `TYPE_ENUM`"""
-        # TODO(xiaozhenliu): convert proto type `TYPE_STRING` to `string`.
+        """Return the proto type constant e.g. `enum`"""
         return WithLocation(
-            FieldDescriptorProto().Type.Name(self.field_pb.type),
+            cast(str, FieldDescriptorProto().Type.Name(self.field_pb.type))[
+                len("TYPE_") :
+            ].lower(),
             self.source_code_locations,
             # FieldDescriptorProto.type has field number 5.
             self.path + (5,),
         )
 
     @property
     def is_primitive_type(self):
-        """Return true if the proto_type is primitive python type like `TYPE_STRING`"""
-        NON_PRIMITIVE_TYPE = ["TYPE_ENUM", "TYPE_MESSAGE", "TYPE_GROUP"]
+        """Return true if the proto_type is primitive python type like `string`"""
+        NON_PRIMITIVE_TYPE = ["enum", "message", "group"]
         return False if self.proto_type.value in NON_PRIMITIVE_TYPE else True
```

Comment:
Nit: while we're changing things, this can be simplified to
```python
return self.proto_type.value not in NON_PRIMITIVE_TYPE
```

Reply:
Good point! Done.